.github/workflows/ci.yml

```diff
@@ -177,24 +179,14 @@ jobs:
 
     strategy:
       matrix:
-        postgres: ['12', '13', '14']
-        postgis: ['2.5', '3.2']
+        postgres: ['14']
```

Comment:
Why don't you use newer Postgres versions for CMIS tests? Are they not supported?

Reply:
Hmm they probably should be, I'll bump it to 17 and check